diff --git a/cmake/dependencies/pybind11.cmake b/cmake/dependencies/pybind11.cmake
index 41c5f0ec1..ea6185ca1 100644
--- a/cmake/dependencies/pybind11.cmake
+++ b/cmake/dependencies/pybind11.cmake
@@ -37,7 +37,7 @@ function(find_pybind11)
             mark_as_advanced(FETCHCONTENT_UPDATES_DISCONNECTED_FETCHEDpybind11)
         endif()
     else()
-        find_package(pybind11 3.0.4 CONFIG REQUIRED)
+        find_package(pybind11 3.0.0 CONFIG REQUIRED)
         message(STATUS "pybind11: Found version '${pybind11_VERSION}'")
     endif()
 endfunction()
